src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
- [ ] Kindly check this test, it's failing.

![image](https://github.com/peter-mwovi/Math-Magician2/assets/93630700/dc8b0e5a-0975-4360-825f-1244cdbf46ff)